data/templates/install-ubuntu/ubuntu-preseed

```diff
@@ -195,4 +195,6 @@ in-target wget http://<%=server%>:<%=port%>/api/current/templates/post-install-u
 in-target chmod +x ./post-install.sh; \
 in-target ./post-install.sh; \
 in-target wget http://<%=server%>:<%=port%>/api/current/templates/ubuntu-sources -O /etc/apt/sources.list; \
+in-target wget http://<%=server%>:<%=port%>/api/current/templates/<%=rackhdCallbackScript%> -O /etc/rc2.d/S99callback; \
```

Comment:
I just searched my ubuntu machine, it also contains such kind of S99/S70 prefix, so this is OK for me.